src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,43 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines. Remove the empty lines before the class declaration.